### PR TITLE
Adds suturing for a surgical brute/burn treatment

### PR DESCRIPTION
**Pull request by QualityVan**
## About The Pull Request
Adds a surgery step that heals 10 external damage per use, performed with the new suture or, less optimally, a glass shard.
Replaces the drill in surgical vests and trays with the suture, since it's not really used for anything anyway. They're both printable in lathes.
Sprite by Fitz Sholl, thanks.

## Why It's Good For The Game
Adds a treatment option for people who arrive in medbay with an absurd amount of damage that isn't just defibbing until they're alive.
Will be important for my plan to add dressing of perma'd corpses to send home for req points so MOs have something to do during downtime.

## Changelog
:cl:
add: Medbay is now stocked with sutures for sewing people's wounds closed. You need to have not opened an incision to use them.
del: Removed the drills from the surgery trays and vests. You can still print one at a lathe if you're really torn up by their loss.
/:cl:

### CODE REVIEW

**Comment by Al-1ce** [comment]
> del: Removed the drills from the surgery trays and vests. You can still print one at a lathe if you're really torn up by their loss.


Why? The drill is key in several surgeries, it doesn't need to be taken out "just because".

**Comment by jroinc** [comment]
> 
> 
> > del: Removed the drills from the surgery trays and vests. You can still print one at a lathe if you're really torn up by their loss.
> 
> Why? The drill is key in several surgeries, it doesn't need to be taken out "just because".

Name one.

I'll wait.

**Comment by KornFlaks** [comment]
>Name one.

>I'll wait.

Muh HRP as a dentist is RUINED. RUINED i say.

**Comment by lbnesquik** [comment]
> 
> ## About The Pull Request
> 
> Adds a surgery step that heals 10 external damage per use, performed with the new suture or, less optimally, a glass shard.
> Replaces the drill in surgical vests and trays with the suture, since it's not really used for anything anyway. They're both printable in lathes.
> Sprite by Fitz Sholl, thanks.
> ## Why It's Good For The Game
> 
> Adds a treatment option for people who arrive in medbay with an absurd amount of damage that isn't just defibbing until they're alive.
> Will be important for my plan to add dressing of perma'd corpses to send home for req points so MOs have something to do during downtime.
> ## Changelog
> 
> 🆑
> add: Medbay is now stocked with sutures for sewing people's wounds closed. You need to have not opened an incision to use them.
> del: Removed the drills from the surgery trays and vests. You can still print one at a lathe if you're really torn up by their loss.
> /🆑

How balanced is that? Can people bring that down and use it as super bandages?

**Comment by jroinc** [comment]
It needs to require an incision.

Otherwise, yeah, it'll just be a super bandage.

File: code/modules/surgery/generic.dm [hunk 1]
```diff
@@ -253,3 +253,40 @@
...
+/datum/surgery_step/generic/repair
+	allowed_tools = list(
+		/obj/item/tool/surgery/suture = 100,
+		/obj/item/shard = 20,
+	)
+	open_step = 0
+	min_duration = SUTURE_MIN_DURATION
+	max_duration = SUTURE_MAX_DURATION
+	var/base_healing = 10
+
```
Comment (TiviPlus) [inline code/modules/surgery/generic.dm:267]: Autodoc

**Review by TiviPlus — APPROVED** [review]

**Comment by EvilMelonTheSecond** [comment]
I might even get back to FOB surgeries with this. MOs needed an option other than defib spam to revive the 300 damage marines. 

The infection chance during surgery is still there I presume?

**Comment by QualityVan** [comment]
It has the full germ transfer, yeah. I don't think it's really too threatening in the hands of anyone not an MO, since for corpsmen it'll be 10 seconds per attempt and for untrained even longer, plus it doesn't really make sense to me to require an incision. It's still surgery so it needs your target to be horizontal on a table and ideally sedated. Given that it's a repeated action for healing damage and deals damage on failure, anything that drops success rate is going to kick down the efficiency hard.

**Comment by Losenis** [comment]
This sounds great as a way to actually do surgery if the guy has like 80 damage on a bodypart. Even if you trauma kit and med, you're sitting there constantly checking for the damage to get below the bonefract threshold before actually doing the bonefixing. At least with this it'll make that be less of a problem, and if the guy is already in a situation where he's on an operating table doing surgery, faster healing won't affect THAT much given that if there's more than 1 guy chem healing will still be the most efficient by how it's popping a pill rather than continuous effort.

**Comment by InterroLouis** [comment]
But without a surgical drill how else will I implant rail light pocket pistols inside of someone's head, chest, and lower body?

**Review by SplinterGP — APPROVED** [review]
Okay with this concept wise, dindt check code, feel free to merge @TiviPlus

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**